alexandria/core/migrations/0015_category_allowed_mime_types.py

```diff
@@ -0,0 +1,79 @@
+            model_name='category',
+            name='allowed_mime_types',
+            field=django.contrib.postgres.fields.ArrayField(base_field=models.CharField(max_length=255), blank=True, null=True, size=None, verbose_name='allowed mime types'),
+        ),
+        migrations.AlterField(
+            model_name='category',
+            name='modified_by_group',
+            field=models.CharField(blank=True, max_length=255, null=True, verbose_name='modified by group'),
+        ),
+        migrations.AlterField(
+            model_name='category',
+            name='modified_by_user',
+            field=models.CharField(blank=True, max_length=150, null=True, verbose_name='modified by user'),
+        ),
+        migrations.AlterField(
+            model_name='document',
+            name='modified_by_group',
+            field=models.CharField(blank=True, max_length=255, null=True, verbose_name='modified by group'),
+        ),
+        migrations.AlterField(
+            model_name='document',
+            name='modified_by_user',
+            field=models.CharField(blank=True, max_length=150, null=True, verbose_name='modified by user'),
+        ),
+        migrations.AlterField(
+            model_name='file',
+            name='modified_by_group',
+            field=models.CharField(blank=True, max_length=255, null=True, verbose_name='modified by group'),
+        ),
+        migrations.AlterField(
+            model_name='file',
+            name='modified_by_user',
+            field=models.CharField(blank=True, max_length=150, null=True, verbose_name='modified by user'),
+        ),
+        migrations.AlterField(
+            model_name='mark',
+            name='modified_by_group',
+            field=models.CharField(blank=True, max_length=255, null=True, verbose_name='modified by group'),
+        ),
+        migrations.AlterField(
+            model_name='mark',
+            name='modified_by_user',
+            field=models.CharField(blank=True, max_length=150, null=True, verbose_name='modified by user'),
+        ),
+        migrations.AlterField(
+            model_name='tag',
+            name='modified_by_group',
+            field=models.CharField(blank=True, max_length=255, null=True, verbose_name='modified by group'),
+        ),
+        migrations.AlterField(
+            model_name='tag',
+            name='modified_by_user',
+            field=models.CharField(blank=True, max_length=150, null=True, verbose_name='modified by user'),
+        ),
+        migrations.AlterField(
+            model_name='tagsynonymgroup',
+            name='modified_by_group',
+            field=models.CharField(blank=True, max_length=255, null=True, verbose_name='modified by group'),
+        ),
+        migrations.AlterField(
+            model_name='tagsynonymgroup',
+            name='modified_by_user',
+            field=models.CharField(blank=True, max_length=150, null=True, verbose_name='modified by user'),
+        ),
```

Comment:
What happened here?

Reply:
`verbose_name` of the `BaseModel` for modified by had a copy paste error. Migrations werent ran to reflect the change of that text.

Reply:
I'd like to see that fixed in a separate commit, even if it implies a separate migration

Reply:
spllit into a seperate commit & migration